commons/src/test/java/com/powsybl/commons/report/ReportNodeTest.java

```diff
@@ -12,15 +12,76 @@
 import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
+import java.nio.file.Path;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * @author Florian Dupuy {@literal <florian.dupuy at rte-france.com>}
  */
 class ReportNodeTest extends AbstractSerDeTest {
 
+    @Test
+    void testValues() throws IOException {
```

Comment:
I do not see any unit test for timestamps. Maybe it is because hard to compare to a reference? Maybe just searching for the right nodes and matching the date with a pattern would be enough?

Reply:
that's more or less what I'm doing in unit test `ReportNodeTest::testTimestamps`. I don't check that the pattern is the right one though. I'll add that!

Reply:
Well I was checking the pattern in fact, but with the default iso-like from java.time.format...